doc/extdev/index.rst

```diff
@@ -187,6 +187,84 @@ as metadata of the extension.  Metadata keys currently recognized are:
   output files can be used when the extension is loaded.  Since extensions
   usually don't negatively influence the process, this defaults to ``True``.
 
+.. _ext-i18n:
+
+Extension internationalization (`i18n`) and localization (`l10n`)
```

Comment:
This page is an index page for extension developers. I think `doc/extdev/i18n.rst` is more appropriate for this topic. Could you move this to the page please?